### PR TITLE
Update for documentation for TF Micro

### PR DESCRIPTION
**Pull request by b0noI**
Proposal includes:
* new line on how to install arm compiler on Mac OS
* fix for the build on Mac OS
* instructions how to train new model in the Cloud with Deep Learning images

### CODE REVIEW

File: tensorflow/lite/experimental/micro/examples/micro_speech/README.md [hunk 1]
```diff
@@ -31,6 +31,8 @@ To understand how TensorFlow Lite does this, you can look at the `TestInvoke()`
 
 Once you have downloaded the dependencies and got the x86/Linux build working, you can try building a version for the STM32F103 'bluepill' device. The following command will build the test and then run it on an emulator, assuming you have Docker installed:
 
+*On Mac OS you need ot have ARM compiler installed, one way of doing so is with brew: brew install caskroom/cask/gcc-arm-embedded*
```
Comment (petewarden) [inline tensorflow/lite/experimental/micro/examples/micro_speech/README.md]: s/ot/to/

Comment (b0noI) [reply tensorflow/lite/experimental/micro/examples/micro_speech/README.md]: done

**Review by petewarden — DISMISSED** [review]
Thanks Slava!

**Comment by b0noI** [comment]
@petewarden thank you for the review, what are the next steps to merge the PR? I do not have permissions to merge this in:(

**Comment by petewarden** [comment]
One of the TF OSS team should pick up the fact that this is ready and merge it in soon, you don't have to do anything. Thanks for putting this together!

**Comment by hgadig** [comment]
@b0noI  Could you resolve the branch conflicts ?

**Comment by hgadig** [comment]
@b0noI  Request you to rebase to resolve the branch conflicts. Once done, we can help to proceed with merging. Appreciate your response.

**Comment by b0noI** [comment]
@petewarden Sorry for delay, will rebase today

**Comment by b0noI** [comment]
done

**Review by petewarden — APPROVED** [review]

**Pull request merged**